src/api/java/com/minecolonies/coremod/colony/requestsystem/location/ILocation.java

```diff
@@ -0,0 +1,31 @@
+package com.minecolonies.coremod.colony.requestsystem.location;
+
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Interface used to describe locations in the world.
+ */
```

Comment:
Colonist are currently unable to switch dimensions, should take care with that, it causes a lot of problems with them atm.